rewrite-core/build.gradle.kts

```diff
@@ -32,6 +32,7 @@ dependencies {
     implementation("io.micrometer:micrometer-core:1.+")
     implementation("io.github.classgraph:classgraph:latest.release")
     implementation("org.yaml:snakeyaml:latest.release")
+    implementation("com.github.albfernandez:juniversalchardet:latest.release")
```

Comment:
This is using the Mozilla public license, would like @jkschneider to weigh in. 

I like this library because it looks reasonably complete and does not have any transitive dependencies, just have a question on the license?

Reply:
1.1 is not apache compatible